include/upipe/ubuf_pic.h

```diff
@@ -388,7 +397,69 @@ static inline int ubuf_pic_blit(struct ubuf *dest, struct ubuf *src,
         int plane_vsize = extract_vsize / src_vsub;
 
         for (int i = 0; i < plane_vsize; i++) {
-            memcpy(dest_buffer, src_buffer, plane_hsize);
+            if (!alpha_plane || threshold == 0) {
+                memcpy(dest_buffer, src_buffer, plane_hsize);
+            } else if (threshold != 0xff) {
+                for (int j = 0; j < plane_hsize; j += 16) {
```

Comment:
you can ask for an aligned buffer in upipe_blit with a urequest flow format, but I don't think ubuf_pic_blit should rely on it.